source/Company.py

```diff
@@ -1,53 +1,184 @@
+# Company Class exercise
+
+def contains_number(value):
+    '''
+    A function which checks if a value includes digits
+    '''
+    if True in [char.isdigit() for char in value]:
+        return True
+    return False
+
+
+def check_string(str: str):
+    '''
+    A function which checks if the conditions of
+    company name are met
+    '''
+    if contains_number(str) or len(str) < 2:
+        return False
+    elif str[0].isupper() and len(str) >= 2:
+        return True
+
+
+def check_int(val):
+    '''
+    A function which checks if a given number is int
+    '''
+    if isinstance(val, int):
+        return True
+    return False
+
+
+def check_positive(num: float):
+    '''
+    A function which returns True if the given number
+    is positive and False otherwise
+    '''
+    if num <= 0:
+        return False
+    return True
+
```

Comment:
I love it that you love functions, but this function is also not needed.
When we did our python training, we did divide simple tasks like this into functions, but now, when the concept is fully understood I want you think upon creating a function: Is it really needed or is it just more extra code.
num > 0 is very understandable, there is no need for check_positive(num).

Reply:
I agree!! This condition does not require a separate function. Ive changed all the parts which involved this function!

Reply:
Good!